common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -14,10 +14,9 @@
 
 define(
 'video/01_initialize.js',
-['video/03_video_player.js', 'video/00_i18n.js'],
-function (VideoPlayer, i18n) {
-    var moment = window.moment;
-
+['video/03_video_player.js', 'video/00_i18n.js', 'moment'],
+function (VideoPlayer, i18n, momentjs) {
+    var moment = momentjs || window.moment;
```

Comment:
@waheedahmed What if `momentjs` is not defined, would it load the video. Because you are expecting to load the moment from window object afterwards.